source/special.tex

```diff
@@ -3203,17 +3203,23 @@
 \end{example}
 
 \pnum
-When the criteria for elision of a copy/move operation are met,
-but not for an \nonterminal{exception-declaration},
-and the object
-to be copied is designated by an lvalue,
-or when the \grammarterm{expression} in a \tcode{return} statement
+In the following copy-initialization contexts, a move constructor might be used instead of a copy constructor:
```

Comment:
Maybe:

  "In the following copy-initialization contexts, a move operation might be used instead of a copy operation:"

We say "A copy operation [...] may be [...] converted to a move operation" in [stmt.return]/2, and mention copy/move operations in [class.copy.elision]/1 and in a few other contexts where we don't necessarily mean a copy/move constructor is used, but do mean a constructor that's constructing from another object of the same type, so this seems like consistent use of terminology (even if we don't define the term).